tests/lib/c_lib/src/test_qsort.c

```diff
@@ -16,7 +16,14 @@ static int compare_ints(const void *a, const void *b)
 	return (aa > bb) - (aa < bb);
 }
 
-void test_qsort(void)
+ZTEST_SUITE(test_c_lib_qsort, NULL, NULL, NULL, NULL, NULL);
```

Comment:
Not needed, tests all go into a single section in the linker so you can actually link the tests here into the same test suite so this PR leaves this 1:1 with the old code, just use `ZTEST(test_c_lib, my_test_name)`